wg-workload-overview/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - sig-operations-leads
```

Comment:
did you mean to write `wg-workload-overview-leads:`?

Reply:
I did